modules/vaos/app/services/vaos/v2/appointments_service.rb

```diff
@@ -72,12 +72,31 @@ def get_appointments(start_date, # rubocop:disable Metrics/ParameterLists
         end
 
         if Flipper.enabled?(:appointments_consolidation, user)
+          eps_before = appointments.select do |appt|
+            appt[:type] == 'epsAppointment' || appt.dig(:provider, :id).present?
+          end.pluck(:id)
+
           filterer = AppointmentsPresentationFilter.new
           appointments.keep_if { |appt| filterer.user_facing?(appt) }
+
+          eps_after = appointments.select do |appt|
+            appt[:type] == 'epsAppointment' || appt.dig(:provider, :id).present?
+          end.pluck(:id)
+          removed = eps_before - eps_after
+
+          removed_msg = removed.any? ? ", removed #{removed}" : ''
+          Rails.logger.info("EPS Debug: Presentation filter kept #{eps_after}#{removed_msg}")
         end
 
         # log count of C&P appointments in the appointments list, per GH#78141
         log_cnp_appt_count(cnp_count) if cnp_count.positive?
+
+        # Log final EPS appointments
+        final_eps = appointments.select do |appt|
+          appt[:type] == 'epsAppointment' || appt.dig(:provider, :id).present?
+        end.pluck(:id)
+        Rails.logger.info("EPS Debug: Final response #{final_eps.any? ? final_eps : 'none'}")
+
```

Comment:
I know that the Rubocop line length enforcement had already been bypassed for this method, but maybe we could break out this new logic into a helper method to keep from bloating it more